Source/Krypton Components/Krypton.Toolkit/Palette Controls/PaletteInputControlContentStates.cs

```diff
@@ -80,12 +82,39 @@ public class PaletteInputControlContentStates : Storage,
         public virtual void PopulateFromBase(PaletteState state)
         {
             // Get the values and set into storage
+            SynchronizeDropDownWidth = false;
             Font = GetContentShortTextFont(state);
             Color1 = GetContentShortTextColor1(state);
             Padding = GetContentPadding(state);
         }
         #endregion
 
+        #region SynchronizeDropDownWidth
+
+        /// <summary>Gets or sets a value indicating whether [synchronize drop down width].</summary>
+        /// <value><c>true</c> if [synchronize drop down width]; otherwise, <c>false</c>.</value>
+        [KryptonPersist(false)]
+        [Category(@"Visuals")]
+        [Description(@"Adjust the drop-down width, based on the controls width.")]
+        [DefaultValue(null)]
+        [AllowNull]
+        public bool SynchronizeDropDownWidth
```

Comment:
Need `Should###` and `Reset###` on these, otherwise they show up as always being bold in the designer

Reply:
@Smurf-IV Implemented, but am I missing something, as it still shows up as bold?

Reply:
You have stated `[DefaultValue(null)]` but it is a normal bool,
So needs to be set as `false` and check accordingly.
I do not see the `Should##` and `Reset###` API's

Reply:
This has now been fixed